src/Util.py

```diff
@@ -17,6 +20,8 @@ def __init__(self):
             ]
         )
         self.spriteCollection["card"] = self.loadCards("./cards/cards_corrected.json")
+        self.spriteCollection["card_conf"] = self.loadCardConf("./cards/cards_corrected.json")
```

Comment:
@rymTooo Is default ["card"] still in use? or we switch to "card_conf" already. Then we should remove old one